index.html

```diff
@@ -32,6 +34,10 @@ <h1 class="appTitle">Film Quote Game</h1>
             </div>
             <div class="play">Play</div>
         </div>
+        <div class="scoreSection">
+            <p class="scoreLabel">Score:  </p>
```

Comment:
You will find that the space at the end of this text is removed in the HTML: it strips out any whitespace at the beginning and end of text automatically.